test/pylib/manager_client.py

```diff
@@ -104,70 +118,91 @@ async def replicas(self) -> int:
         resp = await self.client.get_text("/cluster/replicas")
         return int(resp)
 
-    async def running_servers(self) -> List[str]:
-        """Get list of running servers"""
-        host_list = await self.client.get_text("/cluster/running-servers")
-        return host_list.split(",")
+    async def running_servers(self) -> List[ServerInfo]:
+        """Get dict of host id to IP address of running servers"""
```

Comment:
the comment still mentions `dict` and `host id`, which `ServerInfo` does not contain.

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/0ebda403004c746f8a0019098e9125c160433847 https://github.com/scylladb/scylladb/compare/6635e6163463bf74df6826ee15735d79e6493578..0ebda403004c746f8a0019098e9125c160433847